google-cloud-clients/pom.xml

```diff
@@ -154,16 +154,17 @@
     <site.installationModule>google-cloud-clients</site.installationModule>
     <bom.version>0.77.1-alpha-SNAPSHOT</bom.version><!-- {x-version-update:google-cloud-bom:current} -->
     <api-client.version>1.27.0</api-client.version>
-    <gax.version>1.35.1</gax.version>
+    <!-- make sure to keep gax version in sync with google-cloud-bom -->
+    <gax.version>1.37.0</gax.version>
```

Comment:
As of 1/14/2019, this version of gax is not yet released:
https://github.com/googleapis/gax-java/pull/644